### PR TITLE
03-Make

### PR DESCRIPTION
**Pull request by lapin-v**

### CODE REVIEW

File: 03-Make/soup/Makefile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/make
```
Comment (yekovalyov) [inline 03-Make/soup/Makefile]: Shabang with /usr/bin/make makes command line:
/usr/bin/make ./Makefile
It just use ./Makefile as target. 
If you want use this Makefile as source of rules first line should be
#!/usr/bin/make -f

File: 03-Make/soup/Makefile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	echo "200g meat" >> kitchen/meat.txt
+
+slice_patato: prep_patato check_table
+	mv kitchen/potato.txt kitchen/table/potato_sliced.txt
+	
+prep_patato: check_kitchen
+	echo "500g potato" >> kitchen/potato.txt
+
+clean:
+	find kitchen -type f -name "*.txt" -delete
```
Comment (yekovalyov) [inline 03-Make/soup/Makefile]: Last empty line is missed.

File: 03-Make/soup/check_dirs.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+! #bin/bash
```
Comment (yekovalyov) [inline 03-Make/soup/check_dirs.sh]: What is the meaning of ! #/bin/bash?
Do You mean #!/bin/bash ?

Comment (lapin-v) [reply 03-Make/soup/check_dirs.sh]: Yes )

**Pull request merged**